src/Raven.Client/Http/RavenCommand.cs

```diff
@@ -37,7 +38,8 @@ public interface IRaftCommand
     public abstract class RavenCommand<TResult>
     {
         public CancellationToken CancellationToken = CancellationToken.None;
-        public Dictionary<ServerNode, Exception> FailedNodes;
+
+        public ConcurrentDictionary<ServerNode, Exception> FailedNodes;
```

Comment:
This breaks backward compatibility

Reply:
Why is this concurrent, at all? The command is single threaded, after all.
when we do stuff like read faster, we clone the _command_.